examples/sadtalker/inference.py

```diff
@@ -0,0 +1,90 @@
+import os
+import sys
+import shutil
+from time import strftime
+from addict import Dict
+from utils.arg_parser import parse_args_and_config
+
+args, cfg = parse_args_and_config()
```

Comment:
why initialize it as global variable here?

Reply:
bad habit, modified haha